azure/packages/test/scenario-runner/src/AzureClientRunner.ts

```diff
@@ -33,7 +33,7 @@ export interface AzureClientRunnerConfig {
 }
 export type AzureClientRunnerRunConfig = AzureClientRunnerConfig & IRunConfig;
 
-export class AzureClientRunner extends TypedEventEmitter<IRunnerEvents> implements IRunner {
+export class AzureClientRunner extends CustomEventEmitter<IRunnerEvents> implements IRunner {
```

Comment:
If this is all internal anyway, can we update to the preferred events pattern? and then not expose `CustomEventEmitter`?

Reply:
+1

The preferred pattern is the third example in the CustomEventEmitter doc comment. It'd be neat if we managed to migrate all internal code without ever having to export CustomEventEmitter. Then we could unbless the inheritance pattern - i.e. the first example in the CustomEventEmitter doc comment. I can't say for sure that there isn't some scenario in which doing the inheritance is better in some way than the preferred pattern. But no need to expose it until we discover it (if we ever do).